persistence/src/main/java/edu/unc/lib/dl/persist/services/edit/ExpireEmbargoService.java

```diff
@@ -63,11 +65,13 @@
 
     private static final Timer timer = TimerFactory.createTimerForClass(ExpireEmbargoService.class);
 
+    private static final Logger log = LoggerFactory.getLogger(ExpireEmbargoService.class);
+
     public ExpireEmbargoService() {
     }
 
     // run service every day 1 minute after midnight
-    @Scheduled(cron = "0 1 0 * * *")
+    @Scheduled(cron = "0 15 16 * * *")
```

Comment:
guessing you didn't mean to commit this change

Reply:
nope!  sorry!